### PR TITLE
fix dangling ref to HTTP11

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

**Review by martinthomson — APPROVED** [review]
Looks like this is the new location of the text.

File: draft-ietf-quic-http.md [hunk 1]
```diff
@@ -803,9 +803,9 @@ with the RST bit set.
 
 ## HTTP Upgrade
 
-HTTP/3 does not support the HTTP Upgrade mechanism (Section 9.9 of {{?HTTP11}})
-or 101 (Switching Protocols) informational status code (Section 10.2.2 of
-{{!SEMANTICS}}).
+HTTP/3 does not support the HTTP Upgrade mechanism (Section 6.7 of
+{{?SEMANTICS}) or 101 (Switching Protocols) informational status code (Section
+10.2.2 of {{!SEMANTICS}}).
```
Comment (LPardue) [inline draft-ietf-quic-http.md:808]: Normative and informative in the same para

File: draft-ietf-quic-http.md [hunk 2]
```diff
@@ -803,9 +803,9 @@ with the RST bit set.
 
 ## HTTP Upgrade
 
-HTTP/3 does not support the HTTP Upgrade mechanism (Section 9.9 of {{?HTTP11}})
-or 101 (Switching Protocols) informational status code (Section 10.2.2 of
-{{!SEMANTICS}}).
+HTTP/3 does not support the HTTP Upgrade mechanism (Section 6.7 of
+{{?SEMANTICS}) or 101 (Switching Protocols) informational status code (Section
```
Comment (MikeBishop) [inline draft-ietf-quic-http.md:807]:
```suggestion
{{!SEMANTICS}) or 101 (Switching Protocols) informational status code (Section
```

File: draft-ietf-quic-http.md (hunk 1 above)
Comment (MikeBishop) [reply draft-ietf-quic-http.md:808]: Semantics is normative.

**Pull request merged**